src/components/MovieDialog/MovieDialog.scss

```
@@ -2,149 +2,122 @@
 @import "styles/fonts";
 @import "styles/buttons";
 @import "styles/z-index";
+@import "styles/modal";
 
 .movie-dialog {
-  position: fixed;
-  width: 100%;
-  height: 1788px;
-  left: 0px;
-  top: 0px;
-  z-index: $z-index-modal-background;
-  mix-blend-mode: normal;
-  background: rgba(35, 35, 35, 0.918051);
-
-  &__container {
-    position: fixed;
-    top: 50%;
-    left: 50%;
-    transform: translate(-50%, -50%);
-
-    width: 1000px;
-    height: 800px;
-    z-index: $z-index-modal;
-
-    background: #232323;
-    box-shadow: 0px 0px 2px rgba(0, 0, 0, 0.1), 0px 2px 10px rgba(0, 0, 0, 0.07),
-      0px 10px 20px rgba(0, 0, 0, 0.05), 0px 10px 50px rgba(0, 0, 0, 0.05);
-
-    .close-button {
-      position: absolute;
-      right: 26.12px;
-      top: 27.64px;
+  @include modal-background();
 
+  .modal {
+    @include modal($width: 800px, $height: 700px);
+  }
+  .modal-small {
+    @include modal($width: 700px, $height: 350px);
+  }
+
+  .modal-header {
+    display: flex;
+    flex-direction: column;
+
+    &__button {
+      display: flex;
+      justify-content: flex-end;
+      margin-right: 20px;
+      margin-top: 20px;
```

Please use shorthands

```suggestion
      margin: 0 20px;
```

---

As I had to make some changes with the height of modal I don't need those margins anymore.